arch/arm64/configs/crocodile_defconfig

```diff
@@ -190,21 +190,24 @@ CONFIG_CAN=m
 CONFIG_CAN_RCAR=m
 CONFIG_CAN_RCAR_CANFD=m
 CONFIG_CAN_FLEXCAN=m
-CONFIG_BT=m
+CONFIG_BT=y
+CONFIG_BT_BNEP=y
 CONFIG_BT_HIDP=m
 # CONFIG_BT_HS is not set
 # CONFIG_BT_LE is not set
 CONFIG_BT_LEDS=y
+CONFIG_BT_RFCOMM=y
 # CONFIG_BT_DEBUGFS is not set
 CONFIG_BT_HCIBTUSB=m
-CONFIG_BT_HCIUART=m
+CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART_BCSP=y
 CONFIG_BT_HCIUART_LL=y
 CONFIG_BT_HCIUART_BCM=y
 CONFIG_BT_HCIUART_QCA=y
 CONFIG_CFG80211=m
 CONFIG_MAC80211=m
 CONFIG_MAC80211_LEDS=y
-CONFIG_RFKILL=m
+CONFIG_RFKILL=y
```

Comment:
The description commit message is confusing:
- If you describe the configs, please refer to the help texts of them
- I does "we add the support to the Bluetooth device for MAYA-W1 serials, it was use High-Speed UART" mean?